examples/simple_tabset/app.R

```diff
@@ -7,9 +7,10 @@ ui <- shinyUI(semanticPage(
   tabset(tabs=
            list(
              list(menu="First Tab", content= "Text works well"),
-             list(menu="Second Tab",content=plotOutput("plot1")),
-             list(menu="Second Tab",content=plotlyOutput("plot2"))
-           )
+             list(menu="Second Tab", content=plotOutput("plot1")),
```

Comment:
Please fix styling across this example.
What will happen if we specify in tabset  `active="Second Tab"? It will not work due to lowering menus. Could we also convert active value the same way? I'm starting to think it starts to be more and more messy. WDYT to allow selecting initial tabset only while passing id? I'd prefer that option now.

Reply:
Changed to random if no id added